src/client/pages/Bodem/Bodem.test.tsx

```diff
@@ -126,37 +168,109 @@ const testState: any = {
               status: 'Afgehandeld',
               id: '',
               datePublished: '',
-              isActive: false,
-              isChecked: false,
+              isActive: true,
+              isChecked: true,
             },
           ],
         },
       ],
     },
     status: 'OK',
   },
-};
+} as unknown as AppState;
+
+const createComponent = componentCreator({
+  component: Bodem,
+  routeEntry: AppRoutes.BODEM,
+  routePath: AppRoutes.BODEM,
+});
 
 describe('Bodem', () => {
-  describe('with results', () => {
-    const renderBodem = setupMockApp(Bodem, 'BODEM', testState);
+  describe('With aanvragen', () => {
+    const MockBodem = createComponent(testState);
+
+    describe('Tables', () => {
+      const screen = render(<MockBodem />);
+
+      const lopendeAanvraagTableHeader = screen.getByRole('heading', {
+        name: 'Lopende aanvragen',
+      });
+
+      const afgehandeldeAanvraagTableHeader = screen.getByRole('heading', {
+        name: 'Afgehandelde aanvragen',
+      });
+
+      test('Headers are found', () => {
+        expect(lopendeAanvraagTableHeader).toBeInTheDocument();
+        expect(afgehandeldeAanvraagTableHeader).toBeInTheDocument();
+      });
+
+      const lopendePattern = /(In behandeling)|Ontvangen/;
+      const afgehandeldePattern = /Afgehandeld|Afgewezen/;
+
+      const lopendeAanvraagTable =
+        lopendeAanvraagTableHeader.nextSibling as HTMLElement;
+      if (!lopendeAanvraagTable) {
+        throw Error('No sibling found');
+      }
```

Comment:
Mogelijk wil je hier een assert gebruiken (minder verbose en je test immers voor een programmer error):
```suggestion
      assert(lopendeAanvraagTable, 'lopendeAanvraagTableHeader should have a sibling');
```